packages/app/src/providers/mock/MockProvider.tsx

```diff
@@ -1,18 +1,14 @@
-import type { ReactNode } from 'react';
+import type { PropsWithChildren } from 'react';
 import { useMemo } from 'react';
 
-import Provider from '../Provider';
+import DataProvider from '../DataProvider';
 import { MockApi } from './mock-api';
 
-interface Props {
-  children: ReactNode;
-}
-
-function MockProvider(props: Props) {
+function MockProvider(props: PropsWithChildren<Record<never, never>>) {
```

Comment:
Don't you think it would be clearer to keep the `Props` interface here ?

Reply:
I put back an empty `interface Props {}`